cf_units/tests/unit/unit/test_Unit.py

```diff
@@ -67,7 +67,7 @@ def test_non_gregorian_calendar_conversion_dtype(self):
             (np.float64, True),
             (np.int32, False),
             (np.int64, False),
-            (np.int, False),
+            (int, False),
```

Comment:
See https://numpy.org/doc/stable/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated